drivers/timer/nrf_rtc_timer.c

```diff
@@ -160,7 +166,7 @@ static void set_protected_absolute_ticks(u32_t ticks)
  * it by pointer at runtime, maybe?) so we don't have this leaky
  * symbol.
  */
-void rtc1_nrf_isr(void *arg)
+void rtc_nrf_isr(void *arg)
```

Comment:
Together with this change you'd need to update `tests/arch/arm/arm_irq_vector_table/src/arm_irq_vector_table.c`. See the comment right above this function.